nuget.config

```diff
@@ -9,6 +9,7 @@
     <clear />
     <add key="DevHomeDependencies" value="https://pkgs.dev.azure.com/ms/DevHome/_packaging/DevHomeDependencies/nuget/v3/index.json" />
     <add key="SDKLocalSource" value=".\extensionsdk\_build" />
+    <add key="nuget" value="https://api.nuget.org/v3/index.json" />

```

Comment:
This can't be here, any packages you use need to be added to the DevHomeDependencies feed (talk to me or @EricJohnson327)